packages/core/src/sync-historical/service.ts

```diff
@@ -930,11 +930,12 @@ export class HistoricalSyncService extends Emittery<HistoricalSyncEvents> {
    */
   private _eth_getLogs = async (params: {
     address?: Address | Address[];
-    topics?: LogTopic[];
+    topics: Topics;
```

Comment:
Repeating a bit, but I'd feel better about this type-level `Topics` change if we rename it to `TopicFilter` or `TopicsWithSelector` or something.